<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinWidget</class>
 <widget class="QWidget" name="WinWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>248</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="170">
         <red>255</red>
         <green>207</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="170">
         <red>255</red>
         <green>207</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="saveLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bTimeLabel">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>194</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>黑方用时：</string>
   </property>
  </widget>
  <widget class="QLabel" name="wTimeLabel">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>245</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>白方用时：</string>
   </property>
  </widget>
  <widget class="QLabel" name="bTime">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="wTime">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refuse">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>iconLabel</zorder>
  <zorder>pushButton</zorder>
  <zorder>label</zorder>
  <zorder>saveEdit</zorder>
  <zorder>saveLabel</zorder>
  <zorder>bTimeLabel</zorder>
  <zorder>wTimeLabel</zorder>
  <zorder>bTime</zorder>
  <zorder>wTime</zorder>
  <zorder>refuse</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
